src/applications/lgy/coe/status/components/DocumentList/List.jsx

```diff
@@ -8,30 +8,33 @@ import { formatDateLong } from 'platform/utilities/date';
 import ListItem from './ListItem';
 
 export const formatLabelDate = timestamp =>
-  moment(timestamp).format('MMDDYYYY');
+  moment(timestamp).format('MM-DD-YYYY');
 
-// Example docType: 'pdf', but the mock data has '54'; this may need to be
+// Example documentType: 'pdf', but the local mock data has '54'; this has been
 // updated to extract the file extension from the `mimeType` (e.g. `file.pdf`),
-// which should contain the mime type (expecting `application/pdf`) - awaiting
-// response from LGY
-export const getDocumentType = docType => docType.slice(1).toUpperCase();
+// which really should contain the mime type (expecting `application/pdf`) -
+// still awaiting a response from LGY
+export const getDocumentType = fileName =>
+  fileName
+    .split('.')
+    .pop()
+    .toUpperCase();
 
-export const getDownloadLinkLabel = (timestamp, documentType) =>
-  `Download Notification Letter ${formatLabelDate(
-    timestamp,
-  )} (${getDocumentType(documentType)})`;
+export const getDownloadLinkLabel = timestamp =>
+  `Download Notification Letter ${formatLabelDate(timestamp)}`;
 
 const List = ({ documents }) =>
   documents.map((document, i) => {
-    const { createDate, description, documentType, id } = document;
-    const downloadLinkLabel = getDownloadLinkLabel(createDate, documentType);
+    const { createDate, description, mimeType, id } = document;
```

Comment:
what's a mimeType?

Reply:
LGY doesn't have this named properly. A [proper mime type](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Common_types) for PDFs would be `application/PDF`, but in this case the `mimeType` contains the full file name with extension.